spec/controllers/pages_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe PagesController, type: :feature do
```

Comment:
I believe that it can stay for now. Just some further reading: https://relishapp.com/rspec/rspec-rails/v/5-0/docs/feature-specs/feature-spec